packages/nodejs/src-node_report.cc.patch

```diff
@@ -0,0 +1,12 @@
+--- ./src/node_report.cc.orig	2022-06-04 20:28:45.524998388 +0530
++++ ./src/node_report.cc	2022-06-04 20:30:09.764998356 +0530
+@@ -425,6 +425,9 @@
+     }
+     writer->json_arrayend();
+     uv_free_cpu_info(cpu_info, count);
++  } else {
```

Comment:
This is a hacky fix for test failures of 12 tests in `tests/report/*.js`.

After this only 17 tests will be failing, also in most of these 17 tests, some of them have hard-coded paths, while others don't treat Android as Linux. 

I guess there are a few more tests which are disabled for Android (as the test suite doesn't treat Android as Linux), but should not be done so. I currently don't have any motivation to track down such heavy level of bugs